objgraph.py

```diff
@@ -718,25 +734,28 @@ def show_graph(objs, edge_func, swap_source_target,
         dot.wait()
         if dot.returncode != 0:
             # XXX: shouldn't this go to stderr or a log?
-            print("dot failed to generate '%s' image: output format not supported?")
+            print('dot failed to generate "%s" image:',
+                  'output format not supported?')
```

Comment:
Yup I noticed it too -- I have another pull request to fix this, but I was waiting to push this through. I'll do a quick fix now.